src/drivers/indexeddb.js

```diff
@@ -181,27 +182,164 @@
             }
         }
 
+        // Initialize a singleton container for all running localForages.
+        if (!dbContexts) {
+            dbContexts = {};
+        }
+
+        // Get the current context of the database;
+        var dbContext = dbContexts[dbInfo.name];
+
+        // ...or create a new context.
+        if (!dbContext) {
+            dbContext = {
+                // Running localForages sharing a database.
+                forages: [],
+                // Shared database.
+                db: null
+            };
+            // Register the new context in the global container.
+            dbContexts[dbInfo.name] = dbContext;
+        }
+
+        // Register itself as a running localForage in the current context.
+        dbContext.forages.push(this);
+
+        // Create an array of readiness of the related localForages.
+        var ready = [];
+        for (var j = 0; j < dbContext.forages.length; j++) {
```

Comment:
Because `i` is already used above and jshint complains about that during the tests (with or without `var`). Since I didn't want to change the jshint config, I changed the name. :-)